SQL Query Files/courseQuery.sql

```diff
@@ -0,0 +1,12 @@
+SELECT
```

Comment:
So, regarding queries. I have placed them in a simpler named folder. I will maybe have a way to instead allow for default queries to be called? This would also need some work to catch custom query formatting, like the number of months and such. 

The simplest idea then would just paste a whole query in the .env that is run as-is.

Reply:
I think we can leave the concept of configurable queries for another issue in the next few days.  We can work with the queries in separate files for now.  You raise good questions about handling custom and default queries.  I'll try to make suggestions in a new issue.